### PR TITLE
Tax DB cache

### PR DESCRIPTION
**Pull request by RobinKnipe**
Example artillery config for sending randomised system numbers and names/dates, can be found in the `test/integration/fixtures` folder.

### CODE REVIEW

File: high-load.yml [hunk 1]
```diff
@@ -30,32 +27,31 @@ config:
...
   - name: "System number search"
     flow:
-      - get:
-          url: "/api/v0/events/birth/{{ systemNumber }}"
+    - get:
+        url: "/api/v0/events/birth/{{ systemNumber }}"
+    weight: 1
+  - name: "Random system number search"
+    flow:
+    - function: "pickRandomSystemNumber"
```
Comment (daniel-ac-martin) [inline high-load.yml]: Should this be `getSystemNumber` ?

File: test/integration/fixtures/system-number-config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+config:
+  processor: "../../../src/randomisor.js"
+  target: "http://localhost:3000"
+  phases:
+    - duration: "{{ $processEnvironment.ART_DURATION }}"
+      arrivalRate: "{{ $processEnvironment.ART_RATE }}"
+scenarios:
+  - name: "Random system number search"
+    flow:
+      - function: "getSystemNumber"
```
Comment (daniel-ac-martin) [inline test/integration/fixtures/system-number-config.yml]: Who watches the watchman?!

File: high-load.yml (hunk 1 above)
Comment (RobinKnipe) [reply high-load.yml]: fixed

File: test/integration/fixtures/system-number-config.yml (hunk 2 above)
Comment (RobinKnipe) [reply test/integration/fixtures/system-number-config.yml]: er, I do!?

Comment (RobinKnipe) [reply test/integration/fixtures/system-number-config.yml]: ...no wait, you do!

**Pull request merged**